ethcore/engines/clique/src/tests.rs

```diff
@@ -26,7 +26,7 @@ use ethcore::{
 };
 use engine::Engine;
 use ethereum_types::{Address, H256};
-use ethkey::{Secret, KeyPair};
+use parity_crypto::publickey::{Signature};{Secret, KeyPair};
```

Comment:
This looks odd?

Reply:
Wow! How compiler allowed this....

Reply:
it could be that `{Secret, KeyPair}` are `publicly exported` in the `root` and then it's actually valid